app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -156,56 +172,69 @@ export const SearchPage: React.FC<ISearchType> = ({ showAdvanced }) => {
                                 checked={selected.some((i) => i.id === item.id)}
                               />
                             </Col>
-                            <Col className="tone-date">
-                              <Row>
-                                <Sentiment
-                                  value={item.tonePools?.length ? item.tonePools[0].value : 0}
-                                />
-                                <div className="date text-content">
-                                  {new Date(item.publishedOn).toDateString()}
-                                </div>
-                                <span className="divider"> | </span>
-                                <div className="source text-content">{item.source?.name}</div>
-                                <Show visible={!!item.series?.name}>
-                                  <span className="divider"> | </span>
-                                  <div className="series text-content">{item.series?.name}</div>
-                                </Show>
-                              </Row>
+                            <Col className="sentimentColumn">
+                              <Sentiment
+                                value={item.tonePools?.length ? item.tonePools[0].value : 0}
+                              />
+                            </Col>
+                            <Col className="dateColumn col-date">
+                              <div className="date">{formatDate(item.publishedOn)}</div>
                             </Col>
+                            <Col className="sourceColumn">
+                              {item.contentType === ContentTypeName.AudioVideo
+                                ? item.series?.name
+                                : item.source?.name}
+                            </Col>
+                            <Col className="headlineColumn">
+                              <div className="headline">{item.headline}</div>
+                            </Col>
+                            {(item.mediaType?.name === 'Daily Print' ||
```

Comment:
Hardcoding labels will break when the admin list changes.  To future proof it you should make a component that handles the required number of variations, and one generic for future media types.  Then use an Admin Settings to map media type primary keys to a specific variation.
This way the component isn't tightly linked to media type values in the database.

Reply:
OK, looks like a good approach adding settings. I'll come up with something !
Thank you so much !